python/lsst/ts/linearstage/controllers/igus_dryve.py

```diff
@@ -211,14 +223,14 @@ async def start_mock_ctrl(self):
         )
```

Comment:
Not related to this line but please extract the magic number in line 219 to a constant or use the configured timeout value.

Reply:
Please add a FIXME like you did for other magic numbers.